ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryTableRefreshTest.java

```diff
@@ -81,31 +83,32 @@ public void tableRefreshSingleNode() {
     // Within 1000ms, there should be ~10 packets. But let's be less ambitious and expect at least
     // 5.
     final ArgumentCaptor<Packet> captor = ArgumentCaptor.forClass(Packet.class);
-    for (int i = 0; i < 5; i++) {
+    for (int i = 0; i < 6; i++) {
       timer.runPeriodicHandlers();
+      controller.getRecursivePeerRefreshState().cancelCurrentRound();
+      controller.getPeers().forEach(p -> p.setStatus(PeerDiscoveryStatus.KNOWN));
     }
-    verify(outboundMessageHandler, atLeast(5)).send(eq(peers.get(1)), captor.capture());
-    List<Packet> capturedFindNeighborsPackets =
+    verify(outboundMessageHandler, atLeast(5)).send(any(), captor.capture());
+    final List<Packet> capturedFindNeighborsPackets =
         captor
             .getAllValues()
             .stream()
-            .filter(p -> p.getType().equals(PacketType.FIND_NEIGHBORS))
```

Comment:
that's inaccurate. this test does check what it was originally intended to check, however the mechanism that effectuates the action that it evaluates has changed. the now defunct comment that reads "as the controller performs refreshes, it'll send FIND_NEIGHBORS packets with random target IDs every time" provides a clue, since that doesn't describe the refresh functionality introduced in this pr. the new comment highlights specifically what has changed.